buckets/buckets.go

```diff
@@ -12,6 +15,16 @@ const (
 	SeedName = ".textileseed"
 )
 
+// Role describes an access role for a bucket item.
+type Role int
```

Comment:
Here are the actual access roles. I went with this pattern as opposed to _read_, _write_, _delete_, etc., because those don't imply any relationship between the roles, i.e., _write_ doesn't really imply _read_. However, a writer sounds more clearly like they are also able to read... an admin can do it all, etc.

Reply:
Sounds good.